protocols/migration/migration_reports_4_0_0_to_reports_3_0_0.py

```diff
@@ -63,25 +63,28 @@ def migrate_called_genotype(self, old_genotype):
         new_genotype = self.convert_class(self.new_model.CalledGenotype, old_genotype)
         return self.validate_object(object_to_validate=new_genotype, object_type=self.new_model.CalledGenotype)
 
-    def migrate_report_events(self, old_report_events):
-        return [self.migrate_report_event(old_report_event) for old_report_event in old_report_events]
+    def migrate_report_events(self, old_events):
+        if old_events is None:
+            return None
+        return [self.migrate_report_event(old_event=old_event) for old_event in old_events]
 
-    def migrate_report_event(self, old_report_event):
-        new_report_event = self.convert_class(self.new_model.ReportEvent, old_report_event)
-        new_report_event.genomicFeature = self.migrate_genomic_feature(old_report_event.genomicFeature)
-        if old_report_event.variantClassification is not None:
```

Comment:
we cannot lose this check, if we receive a null variant classification it will fail

Reply:
<discussed IRL>